### ISSUE TITLE
Strange behavior with comma separated parameter

### ISSUE THREAD

**Issue by kot13** [livewire/livewire#517]
**Describe the bug**
Strange behavior with comma separated parameter

**To Reproduce**
1) Install latest Laravel and latest livewire (0.5.3)
2) Create livewire component:
`$ php artisan make:livewire test`
3) Update view and component class:
resources/views/livewire/test.blade.php:
```
<div>
    @php
    $text = 'test, test';
    @endphp
    <button wire:click="test('{{ $text }}')">Test</button>
</div>
```

app/Http/Livewire/Test.php
```
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class Test extends Component
{
    private $text;

    public function render()
    {
        return view('livewire.test');
    }

    public function test($text)
    {
        $this->text = $text;
    }
}
```
**Expected behavior**
In browser console:
```
SyntaxError: '' literal not terminated before end of script
```

But if you change the view to (delete one space in `$text`):
```
<div>
    @php
    $text = 'test,test';
    @endphp
    <button wire:click="test('{{ $text }}')">Test</button>
</div>
```
Everything works fine. 

**Desktop (please complete the following information):**
 - Browser is any: FF, Chrome, Safari and etc.

**Comment by nuernbergerA** [comment]
I will open a PR to fix this after #541 is fixed

**Comment by calebporzio** [comment]
Should be fixed in the next release. Thank you @kot13 for the thorough issue writeup, and thank you @nuernbergerA for the PR!

**Comment by nuernbergerA** [comment]
@calebporzio i think its still not fixed.
I'll open a new PR to handle this case.

**Comment by kot13** [comment]
I updated to 0.7.1
Unfortunately bug is not fixed.
